src/python/pants/backend/python/goals/export_integration_test.py

```diff
@@ -1,102 +1,112 @@
+
+    export_prefix = os.path.join("dist", "export", "python", "virtualenvs")
+    py_minor_version = f"{platform.python_version_tuple()[0]}.{platform.python_version_tuple()[1]}"
+    for resolve, ansicolors_version in [("a", "1.1.8"), ("b", "1.0.2")]:
+        export_dir = os.path.join(export_prefix, resolve, platform.python_version())
+        assert os.path.isdir(export_dir), f"expected export dir '{export_dir}' does not exist"
+
+        lib_dir = os.path.join(export_dir, "lib", f"python{py_minor_version}", "site-packages")
+        expected_ansicolors_dir = os.path.join(
+            lib_dir, f"ansicolors-{ansicolors_version}.dist-info"
         )
-        targets = rule_runner.request(Targets, [AddressSpecs([DescendantAddresses("src/foo")])])
-        all_results = rule_runner.request(ExportResults, [ExportVenvsRequest(targets)])
-
-        for result in all_results:
-            assert len(result.post_processing_cmds) == 2
-
-            ppc0 = result.post_processing_cmds[0]
-            assert ppc0.argv == (
-                os.path.join("{digest_root}", ".", "pex"),
-                os.path.join("{digest_root}", "requirements.pex"),
-                "venv",
-                "--pip",
-                "--collisions-ok",
-                "--remove=all",
-                f"{{digest_root}}/{current_interpreter}",
-            )
-            assert ppc0.extra_env == FrozenDict({"PEX_MODULE": "pex.tools"})
-
-            ppc1 = result.post_processing_cmds[1]
-            assert ppc1.argv == (
-                "rm",
-                "-f",
-                os.path.join("{digest_root}", ".", "pex"),
-            )
-            assert ppc1.extra_env == FrozenDict()
-
-        return all_results
-
-    resolve_results = run(enable_resolves=True)
-    assert len(resolve_results) == 2
-    assert {result.reldir for result in resolve_results} == {
-        "python/virtualenvs/a",
-        "python/virtualenvs/b",
-    }
+        assert os.path.isdir(
+            expected_ansicolors_dir
+        ), f"expected dist-info for ansicolors '{expected_ansicolors_dir}' does not exist"
 
-    no_resolve_results = run(enable_resolves=False)
-    assert len(no_resolve_results) == 1
-    assert no_resolve_results[0].reldir == "python/virtualenv"
+    for tool_config in EXPORTED_TOOLS:
+        export_dir = os.path.join(export_prefix, "tools", tool_config.name)
+        assert os.path.isdir(export_dir), f"expected export dir '{export_dir}' does not exist"
+
+        # Wanted to check the output of calling each tool pex with `--version`, but not all tools
+        # implement that flag.
+        lib_dir = os.path.join(export_dir, "lib", f"python{py_minor_version}", "site-packages")
+        expected_tool_dir = os.path.join(
+            lib_dir, f"{tool_config.name}-{tool_config.version}.dist-info"
+        )
+        assert os.path.isdir(
+            expected_tool_dir
+        ), f"expected dist-info for {tool_config.name} '{expected_tool_dir}' does not exist"
```

Comment:
TODO: Add some plugins / extra requirements to the tools that support it, and add assertions here checking that everything gets installed.